scalameta/src/main/scala/scala/meta/internal/parsers/Parsers.scala

```diff
@@ -634,8 +644,8 @@ private[meta] class Parser(val input: Input)(implicit val dialect: Dialect) { pa
     ts.toList
   }
 
-  @inline final def commaSeparated[T <: Tree : AstMetadata](part: => T): List[T] =
-    tokenSeparated[`,`, T](sepFirst = false, part)
+  @inline final def commaSeparated[T <: Tree : AstMetadata](part: => T, useUnquoteForEllipsis: Boolean = false): List[T] =
+    tokenSeparated[`,`, T](sepFirst = false, part, useUnquoteForEllipsis)
```

Comment:
In this method https://github.com/Dveim/scalameta/blob/master/scalameta/src/main/scala/scala/meta/internal/parsers/Parsers.scala#L2220-2220.

Here (https://github.com/Dveim/scalameta/blob/master/scalameta/src/main/scala/scala/meta/internal/parsers/Parsers.scala#L2251) it would parse ellipsis as mods, what is wrong in this case.